src/finn/custom_op/fpgadataflow/matrixvectoractivation.py

```diff
@@ -122,10 +133,14 @@ def get_nodeattr_types(self):
             # vector through the accelerator. This will get rid of any old
             # weight data from the weight FIFOs.
             "runtime_writeable_weights": ("i", False, 0, {0, 1}),
-        }
+            "preferred_impl_style" : ("s", False, "hls", {"hls", "rtl"}),
```

Comment:
This node attribute doesn't need to be added here, when executing `my_attrs.update(super().get_nodeattr_types())`, it is automatically inherited from HWCustomOp. Please remove this. In general, when reviewing the node attributes, the HW abstraction layer for the `MatrixVectorActivation` should ideally only contain node attributes that the HLS and RTL variant of the MVAU share. Please ensure that this is the case. HLS- or RTL-specific attributes can be added in the _hls or _rtl node variants, like here: https://github.com/Xilinx/finn/blob/refactor/rtl_integration/src/finn/custom_op/fpgadataflow/rtl/streamingfifo_rtl.py#L50

Reply:
Good point, I've removed it now. Thanks!